tests/edge-to-edge/server/app.spec.ts

```diff
@@ -5,6 +5,15 @@ import nock from 'nock';
 import { config } from '@/server/TermboxConfig';
 import 'jest-dom/extend-expect';
 import HttpStatus from 'http-status-codes';
+import Vue from 'vue';
```

Comment:
Wouldn't `tests/config/setup.ts` be a better place for this?

Reply:
In contrast to real HTTP request hitting express (inside of which env is pretty much empty, i.e. not in `NODE_ENV: 'development'` in particular), the server-side code of the edge-to-edge tests is executed with the local env, i.e. development mode, which triggers the warnings inside vue(x).